open-craft/opencraft
# BB-1774 Don't email unverified

## Description

The changes in this PR prevents sending an email informing a redeployment is necessary when an unverified user changes instance settings. An AppServer will be deployed automatically when the user confirms his/her email.

## Testing

1. Checkout this branch on Stage.
1. Create an user and instance, but don't verify the account.
1. Modify the instance parameters.
1. Make sure the message indicates the need for verifying the email.
1. Verify the email.
1. Modify the instance parameters.
1. Make sure the message mentions the need for redeployment.
